src/index.html

```diff
@@ -2,11 +2,325 @@
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <span>
+        <a href="#home">
+          <img
+          src="./images/logo.png"
+          alt="MOYO logo"
+          class="header__logo"
+        >
+        </a>
+      </span>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#apple">APPLE</a></li>
+          <li class="nav__item"><a class="nav__link" href="#samsung">SAMSUNG</a></li>
+          <li class="nav__item"><a class="nav__link" href="#smartphones">СМАРТФОНЫ И ТЕЛЕФОНЫ</a></li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops"
+            >НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item"><a class="nav__link" href="#gadgets">ГАДЖЕТЫ</a></li>
+          <li class="nav__item"><a class="nav__link" href="#tablets">ПЛАНШЕТЫ</a></li>
+          <li class="nav__item"><a class="nav__link" href="#photo">ФОТО</a></li>
+          <li class="nav__item"><a class="nav__link" href="#video">ВИДЕО</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="product">
+        <div class="product__cards">
+
+          <div class="product-card" data-qa="card">
+            <img
+              src="/images/imac.jpeg"
+              alt="iMac picture"
+              class="product-card__image"
+            >
+
+            <div class="product-card__data">
+              <h1 class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+              <div class="product-card__item-code">Код товара: 195434</div>
+            </div>
+
+            <div class="product-card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <span class="product-card__reviews-count">Отзывов: 5</span>
+            </div>
+
+            <div class="product-card__price">
+              <div class="product-card__price-title">Цена:</div>
+              <div class="product-card__price-uah">69 999 грн</div>
+            </div>
+
+            <a
+            href="#toBuy"
+            data-qa="card-hover"
+            class="product-card__button"
+            >
```

Comment:
Just fix indentation here
```suggestion
            <a
              href="#toBuy"
              data-qa="card-hover"
              class="product-card__button"
            >
```